server/src/helpers/generateNumbers.js

```diff
@@ -1,7 +1,13 @@
 import db from '../dummyDb/db';
 
 const { accounts } = db;
-
+/**
```

Comment:
Good job adding a JSDoc. However, this function does not take in any param. Please update it in your next PR. It also doesn't return boolean